security-groups.tf

```diff
@@ -35,6 +35,24 @@ resource "aws_security_group" "nat" {
   }
 }
 
+/* Security group for the Gandalf server */
+resource "aws_security_group" "gandalf" {
+  name = "tsuru-gandalf"
+  description = "Security group for Gandalf instance that allows SSH access from internet"
+  vpc_id = "${aws_vpc.default.id}"
+
+  ingress {
+    from_port = 22
+    to_port   = 22
+    protocol  = "tcp"
+    cidr_blocks = ["80.194.77.0/24"]
```

Comment:
Luigi has noted that this is consistent with the other entries in this file. So we're going to merge as-is and tackle them in a separate PR.